package.json

```diff
@@ -2,117 +2,86 @@
-    "docs": "npm-run-all docs:*",
+    "postclean": "mkdirp dist",
+    "docs": "run-p docs:*",
     "docs:api": "jsdoc src -r -d docs/api",
     "docs:toc": "doctoc README.md",
     "lint": "vjsstandard",
-    "start": "npm-run-all -p watch start:*",
-    "start:serve": "babel-node scripts/server.js",
-    "pretest": "npm-run-all lint build",
+    "prestart": "npm run build",
+    "start": "run-p start:server watch",
+    "start:server": "node scripts/server.js",
+    "pretest": "run-s lint build:test:js",
     "test": "karma start test/karma.conf.js",
-    "preversion": "npm test",
-    "version": "npm run build",
-    "watch": "npm-run-all -p watch:*",
-    "watch:docs": "nodemon --watch src/ --exec npm run docs",
-    "watch:js": "npm-run-all -p watch:js:babel watch:js:browserify watch:js:switcher",
-    "watch:js:babel": "npm run build:js:babel -- --watch",
-    "watch:js:browserify": "watchify . -v -g browserify-shim -o dist/videojs-http-streaming.js",
-    "watch:js:switcher": "watchify utils/switcher/switcher.js -v -t babelify -g browserify-shim -o dist/switcher.js",
-    "watch:test:browserify": "watchify test/browserify-test.js -o dist-test/browserify-test.js",
-    "watch:test:webpack": "watchify test/webpack-test.js -o dist-test/webpack-test.js",
-    "watch:test": "npm-run-all -p watch:test:manifest watch:test:js watch:test:webpack watch:test:browserify",
-    "watch:test:js": "node scripts/watch-test.js",
+    "prebuild:test:js": "run-s build:webworker build:test:manifest",
+    "build:test:js": "rollup -c scripts/test.rollup.config.js",
+    "build:test:manifest": "node -e \"var b=require('./scripts/manifest-data.js'); b.build();\"",
+    "watch": "run-p watch:js watch:test:manifest watch:test:js watch:js:switcher",
+    "watch:js": "rollup -c scripts/rollup.config.js -w",
+    "watch:js:switcher": "rollup -c scripts/switcher.rollup.config.js -w",
+    "watch:test:js": "rollup -c scripts/test.rollup.config.js -w",
     "watch:test:manifest": "node -e \"var b=require('./scripts/manifest-data.js'); b.watch();\"",
-    "prepublish": "not-in-install && npm run build || in-install"
+    "preversion": "npm test",
+    "version": "node scripts/version.js",
+    "prepublish": "in-publish && npm run build || not-in-publish"
   },
   "keywords": [
     "videojs",
     "videojs-plugin"
   ],
   "author": "Brightcove, Inc",
   "license": "Apache-2.0",
-  "browserify": {
-    "transform": [
-      "browserify-versionify"
-    ]
-  },
-  "browserify-shim": {
-    "qunit": "global:QUnit",
-    "sinon": "global:sinon",
-    "video.js": "global:videojs"
-  },
   "vjsstandard": {
     "ignore": [
       "dist",
       "dist-test",
       "docs",
-      "es5",
       "test/karma.conf.js",
       "scripts",
       "utils",
       "test/test-manifests.js"
     ]
   },
   "files": [
-    "dist/",
-    "es5/"
+    "dist/"
   ],
   "dependencies": {
     "aes-decrypter": "1.0.3",
```

Comment:
we should upgrade to aes-decrypter 3 at some point, it is rolluped. I think/hope that 3 has the correct decrypter stuff and not the broken version that 2.0 had.